mkdocs.yml

```diff
@@ -331,6 +331,7 @@ nav:
         - Add remote PIM support: pim/add_remote_pim_support.md
     - Commerce:
         - Commerce: commerce/commerce.md
+        - Interactive demo test: commerce/interactive_demo.md
```

Comment:
Embedding the full demo doesn't work for me, it covers navigation and breaks the page. I suppose we should remove this entry and the test file?

Reply:
Yep, I see that now, I didn't know it when writing this comment 😄